sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -140,6 +141,7 @@ def tune(
         parameters: Dict[str, Any],
         base_image: str = constants.BASE_IMAGE_TENSORFLOW,
         namespace: Optional[str] = None,
+        env_per_trial: Union[Dict[str, str], List[Union[V1EnvVar, V1EnvFromSource]], None] = None,
```

Comment:
Should we make this type Optional and import libs from Kubernetes client, similar to [other Kubernetes modules](https://github.com/kubeflow/katib/blob/bab4e4d63a4a61340a749b54c809e2d71af78a3a/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L351) ?
```suggestion
        env_per_trial: Optional[Union[Dict[str, str], List[Union[client.V1EnvVar, client.V1EnvFromSource]]]] = None,
```

Reply:
Thanks for the great suggestion, I wasn't aware that those `client.models` are also importable from the `client` level!